### PR TITLE
changes for review 1

### PR DESCRIPTION
**Pull request by asiddi24**
- Changed flowchart placement
- Edited Sciserver doc
- Edited installation doc

### CODE REVIEW

File: docs/quick.rst [hunk 1]
```diff
@@ -94,4 +94,9 @@ Click :ref:`datasets` for a list of datasets available on SciServer.
 
 Check out `Tutorial <Tutorial.ipynb#Tutorial>`_, Use Cases, and :ref:`api` to learn more about OceanSpy and its features, and feel free to open an `issue here <https://github.com/malmans2/oceanspy/issues>`_, or to send an email to `mattia.almansi@jhu.edu <mattia.almansi@jhu.edu>`_ if you have any questions.
 
+The flowchart below is meant to be a guiding post to understand the way OceanSpy is designed to be used by the oceanographic community.
```
Comment (ThomasHaine) [inline docs/quick.rst]: Reword to:

The flowchart below shows how OceanSpy is designed to be used by the oceanographic community.

**Review by ThomasHaine — APPROVED** [review]
Looks good. I reworded the flowchart text.

File: docs/quick.rst (hunk 1 above)
Comment (asiddi24) [reply docs/quick.rst]: done.

**Comment by malmans2** [comment]
@asiddi24 @ThomasHaine I made a few changes:
1. Clarify "statement of need" in the front page
2. Improve dependencies list and mention test in the installation page
3. Add a few comments in the tutorial/use cases: all cells now are preceded by a comment
4. For the numerical methods comment, I think we are already in good shape. Gradient, divergence, curl, .... are "home-made" and use the same numerical method of the MITgcm (finite volume). The reference at the bottom of these functions used to be  `MITgcm: link`, and now is `Numerical Method: link`. Smart-name functions have an explicit link to the numerical method used by the MITgcm (e.g., `curl`), while Smart-name functions used by Fixed-name functions are listed under `See Also` (e.g., `curl` is listed in the documentation of the `relative_vorticity` function).

Feel free to change anything. I'll wait for your OK to merge and respond to the reviewer.

**Review by asiddi24 — COMMENTED** [review]
Its coming along better now

**Review by asiddi24 — COMMENTED** [review]
installation and tests and statement of need are good to go from my side

**Review by hooteoos-waltz — COMMENTED** [review]
The changes sound good me too. Just a minor comment on the flow chart: Either change the name of flow chart to road map, or use actual flowchart shapes: oval for end/begin, rectangle for process, diamond for decisions, etc.

**Comment by malmans2** [comment]
@ThomasHaine can you please review this: https://github.com/asiddi24/oceanspy/blob/review_changes_1/README.rst
It's the new home page of both github and readthedocs.

**Comment by ThomasHaine** [comment]
Looks good. Maybe qualify the first sentence a little where it says “analyze and visualize oceanographic data sets”. OceanSpy works well for ocean model datasets, less obviously for observational data sets.



On Jun 19, 2019, at 1:16 PM, Mattia Almansi <notifications@github.com<mailto:notifications@github.com>> wrote:


@ThomasHaine<https://github.com/ThomasHaine> can you please review this: https://github.com/asiddi24/oceanspy/blob/review_changes_1/README.rst
It's the new home page of both github and readthedocs.

**Pull request merged**